src/mixins/view.js

```diff
@@ -205,7 +206,7 @@ const ViewMixin = {
   triggerMethod() {
     const ret = triggerMethod.apply(this, arguments);
 
-    this._triggerEventOnBehaviors(arguments);
+    this._triggerEventOnBehaviors.apply(this, arguments);
```

Comment:
I'll take another 🔪  at the perf diffs, I sort of remember it having to do with bad context warning via profiling, but I think that is unavoidable.